build.gradle

```diff
@@ -67,6 +67,7 @@ subprojects {
     apply plugin: 'java'
     apply plugin: 'maven'
     apply plugin: 'signing'
+    apply plugin: 'maven-publish'
```

Comment:
What does this plugin do?

I thought it was (only) for the previous (non-working) approach at publishing to sonatype... so when the approach was changed, I removed it.